stubs/astropy/METADATA.toml

```diff
@@ -0,0 +1,4 @@
+version = "5.2.*"
+
+[tool.stubtest]
+ignore_missing_stub = false
```

Comment:
If there are currently things missing from these stubs, you'll want to have this set to `true` for now, or you'll get a barrage of complaints from stubtest about missing symbols :)